### DISCUSSION TITLE
Database filename and directory display

### DISCUSSION THREAD

**Discussion by Peterl777** [pawelsalawa/sqlitestudio#4186] [Q&A]
### Details
I'd like it to be easier to find details for the current database, most importantly the actual file that it is stored in. I did (eventually) find it under Database> Edit the database. But maybe if it was easier to find? Maybe click on the database name (on the list) and it show somewhere?

This has come up as I have more than one copy of the database, with the same filename, but in different directories. I found it difficult to check which one I was using.

### SQLiteStudio version
3.3.3

**Comment by pawelsalawa** [comment]
Hold your mouse over the database. There will be a popup hint with details.
